cli/internal/cmd/apply.go

```diff
@@ -807,18 +806,18 @@ type warnLogger struct {
 
 // Infof messages are reduced to debug messages, since we don't want
 // the extra info when using the CLI without setting the debug flag.
-func (wl warnLogger) Infof(fmtStr string, args ...any) {
-	wl.log.Debugf(fmtStr, args...)
+func (wl warnLogger) Info(fmtStr string, args ...any) {
+	wl.log.Debug(fmtStr, args...)
 }
 
 // Warnf prints a formatted warning from the validator.
-func (wl warnLogger) Warnf(fmtStr string, args ...any) {
+func (wl warnLogger) Warn(fmtStr string, args ...any) {
 	wl.cmd.PrintErrf("Warning: %s\n", fmt.Sprintf(fmtStr, args...))
```

Comment:
```suggestion
func (wl warnLogger) Warn(msg string, args ...any) {
	wl.cmd.PrintErrf("Warning: %s\n", fmt.Sprintf(fmtStr, args...))
```
this also needs some code for printing correctly as args are not format string values anymore